Realm.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
   s.osx.vendored_library    = 'core/libtightdb.a'
 
   s.subspec 'Headers' do |s|
-    s.source_files          = 'include/**/*.h'
+    s.source_files          = 'include/**/*.{h,hpp}'
```

Comment:
It makes the podspec more complicated, not less (at least assuming that the explicit `private_header_files` isn't pointless) and doesn't have any benefits.